share/spice/earthquakes/earthquakes.js

```diff
@@ -0,0 +1,49 @@
+(function (env) {
+    "use strict";
+
+    env.ddg_spice_earthquakes = function(api_result) {
+
+        // Validate the response (customize for your Spice)
+        if (!api_result || !api_result.metadata || api_result.metadata.count == 0 || api_result.features.length == 0) {
+            return Spice.failed('earthquakes');
+        }
+
+        // Render the response
+        Spice.add({
```

Comment:
This needs to be wrapped in `DDG.require("maps", function(){ ... });` for the map tiles to load